lib/llm/message.rb

```diff
@@ -20,6 +20,13 @@ def logprobs
       @extra[:logprobs]
     end
 
+    ##
```

Comment:
Can we link to some relevant docs - similar to `logprobs` ? Although there's more providers to cover this time

Reply:
Yes! Thanks.

Reply:
@0x1eef should we just link to `stop_reason` from OpenAI? or link all of them? There's probably no generic docs for it

Reply:
If we could link to the relevant docs for each I think that'd be helpful - failing that, let's just reference OpenAI